serialzy/serializers/exception.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import sys
+from inspect import isclass
+from typing import Any, BinaryIO, Callable, Dict, Type, Union, Optional
+
+from packaging import version  # type: ignore
+from serialzy.api import Schema, VersionBoundary
+from serialzy.base import DefaultSchemaSerializerByValue
+from serialzy.utils import cached_installed_packages
+
+_LOG = logging.getLogger(__name__)
+
+
+class ExceptionSerializer(DefaultSchemaSerializerByValue):
+    DATA_FORMAT = "raw_exception"
+
+    def _serialize(self, obj: BaseException, dest: BinaryIO) -> None:
+        from tblib import pickling_support  # type: ignore
+        pickling_support.install()
+        import pickle  # type: ignore
+        pickle.dump(sys.exc_info(), dest)
+
+    def _deserialize(self, source: BinaryIO, schema_type: Type, user_type: Optional[Type] = None) -> Any:
+        self._check_types_valid(schema_type, user_type)
+        import pickle  # type: ignore
+        return pickle.load(source)
+
+    def supported_types(self) -> Union[Type, Callable[[Type], bool]]:
+        def supported(x):
+            return isclass(x) and issubclass(x, BaseException)
+        return supported
+
+    def available(self) -> bool:
+        base_available = super().available()
```

Comment:
1) you should move tblib requirement from lzy to serialzy.
2) if you have this requirement, how it can be unavailable?